docs/template_plugin/tests/functional/op_reference/roi_pooling.cpp

```diff
@@ -0,0 +1,131 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <tuple>
+
+#include "base_reference_test.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+
+struct ROIPoolingParams {
+    ROIPoolingParams(const size_t iH, const size_t iW, const size_t ch, const size_t rois,
+                     const size_t oH, const size_t oW, const float sS, const std::string mode,
+                     void inputInit(float&),
+                     const std::vector<float>& proposalValues, const std::vector<float>& outputValues,
+                     size_t iSize = 0, size_t pSize = 0, size_t oSize = 0)
```

Comment:
Looks like you don't use `iSize`, `pSize` and etc. please remove all redundant parameters

Reply:
Done